src/engines-experimental/caching.cc

```diff
@@ -57,9 +57,35 @@ namespace kv
 
 caching::caching(std::unique_ptr<internal::config> cfg)
 {
-	if (!readConfig(*cfg))
-		throw std::runtime_error(
-			"caching Exception"); // todo propagate start exceptions properly
+	auto &config = *cfg;
+
+	std::string subEngine;
+	getString(config, "subengine", subEngine);
+
+	getString(config, "remote_type", remoteType);
+	getString(config, "remote_user", remoteUser);
+	getString(config, "remote_pwd", remotePasswd);
+	getString(config, "remote_url", remoteUrl);
```

Comment:
what's the difference between host and remote_url ? it seems that the second one is never used